README.rst

```diff
@@ -51,7 +51,9 @@ To install the latest development version run:
 
 .. code-block:: bash
 
-    $ pip3 install coala-quickstart --pre
+    $ git clone https://github.com/coala/coala-quickstart
+    $ cd coala-quickstart
+    $ pip3 install . 
```

Comment:
Add the editable `-e` flag too, and add one line explaining why it's convenient.